lib/core-integration/src/Test/Integration/Scenario/API/Transactions.hs

```diff
@@ -1474,6 +1491,91 @@ spec = do
               txs2 <- listTransactions ctx w (Just te) (Just te) Nothing
               length <$> [txs1, txs2] `shouldSatisfy` all (== 0)
 
+    it "TRANS_DELETE_01 - Single Output Transaction" $ \ctx -> do
+        (wa, wb) <- (,) <$> fixtureWallet ctx <*> fixtureWallet ctx
+        addrs <- listAddresses ctx wb
+
+        let amt = 1
+        let destination = (addrs !! 1) ^. #id
+        let payload = Json [json|{
+                "payments": [{
+                    "address": #{destination},
+                    "amount": {
+                        "quantity": #{amt},
+                        "unit": "lovelace"
+                    }
+                }],
+                "passphrase": "cardano-wallet"
+            }|]
+        let (feeMin, feeMax) = ctx ^. feeEstimator $ TxDescription
+                { nInputs = 1
+                , nOutputs = 1
+                }
+
+        r <- request @(ApiTransaction t) ctx (postTxEp wa) Default payload
+        verify r
+            [ expectSuccess
+            , expectResponseCode HTTP.status202
+            , expectFieldBetween DSL.amount (feeMin + amt, feeMax + amt)
+            , expectFieldEqual DSL.direction Outgoing
+            , expectFieldEqual DSL.status Pending
+            ]
+
+        let (ApiTransaction txId _ _ _ _ _ _ _ _) = getFromResponse Prelude.id r
+
+        ra <- request @ApiWallet ctx (getWalletEp wa) Default Empty
+        verify ra
+            [ expectSuccess
+            , expectFieldBetween balanceTotal
+                ( faucetAmt - feeMax - amt
+                , faucetAmt - feeMin - amt
+                )
+            , expectFieldEqual balanceAvailable (faucetAmt - faucetUtxoAmt)
+            ]
```

Comment:
Same as above, the check for fee superfluous for this scenario :+1: 
It'll be sufficient to check that the `balanceAvailable < balanceTotal` which means that some funds are locked in a transaction.

Reply:
done